pancancer_utilities/scripts/classify_cancer_type.py

```diff
@@ -41,13 +46,34 @@
 p.add_argument('--seed', type=int, default=cfg.default_seed)
 p.add_argument('--shuffle_labels', action='store_true',
                help='Include flag to shuffle labels as a negative control')
+p.add_argument('--subset_mad_genes', type=int, default=-1,
+               help='If included, subset gene features to this number of\
+                     features having highest mean absolute deviation.')
 p.add_argument('--verbose', action='store_true')
 args = p.parse_args()
 
 np.random.seed(args.seed)
 if args.verbose:
     logging.basicConfig(level=logging.DEBUG, format='%(message)s')
 
+# create directory for the gene
+dirname = 'pancancer' if args.use_pancancer else 'single_cancer'
+gene_dir = os.path.join(args.results_dir, dirname, args.gene)
```

Comment:
Another minor suggestion. Thanks to @danich1 I now use `pathlib` a lot.
```suggestion
gene_dir = Path(args.results_dir, dirname, args.gene).resolve()
```